config/database.yml

```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: utf8
+  database: project_development
+  pool: 5
+  username: vladimir
```

Comment:
this mean that I need the same username, change it to <%= ENV["WIZZART_USERNAME %> and set env variable on your pc